src/org/labkey/test/WebDriverWrapper.java

```diff
@@ -3189,6 +3192,41 @@ private void setHtml5DateInput(WebElement el, Date date)
         el.sendKeys(formDate);
     }
 
+    private void setHtml5NumberInput(WebElement el, String text)
+    {
+
+        el.clear();
+
+        // Calling clear on the element doesn't reliably clear the field, if it fails try the hack.
+        String valueAsNumber = el.getAttribute("valueAsNumber").trim().toLowerCase();
+        if(!valueAsNumber.equals("nan"))
+        {
+
+            int length = valueAsNumber.length();
+
+            // Double click to edit the cell.
+            Actions actions = new Actions(getDriver());
+            actions.doubleClick(el).perform();
+
+            log("WARNING: Clear did not remove the text, going to clear the field manually.");
```

Comment:
There's no need to warn about this. We're just dealing with expected behavior. A warning implies that there is some better method that would work better.

Reply:
Fixed.